src/lib.rs

```diff
@@ -256,6 +256,22 @@ impl<T> Event<T> {
     ///
     /// In certain cases, this function emits a `SeqCst` fence before notifying listeners.
     ///
+    /// ## Return Value
+    ///
+    /// This function returns the number of [`EventListener`]s that were notified by this
+    /// call. However, in some cases, the notification will be delayed and this function will
+    /// return `0` instead of the number of listeners actually notified. Therefore, the return
+    /// value of this function should not be relied upon for soundness and should be used only
+    /// as a hint.
+    ///
+    /// The cases where `0` will be spuriously returned include:
+    ///
+    /// - If the `std` feature is disabled, spurious returns will happen under high contention,
+    ///   such as when another thread is taking a while to `notify` the event.
+    /// - If the `std` feature is enabled, no spurious returns are possible, since the `std`
+    ///   implementation uses system locking primitives to ensure there is no unavoidable
+    ///   contention.
```

Comment:
Maybe I wasn't very clear, sorry. This list item is about when spurious return is impossible when this is supposed to be list of cases of spurious returns. Looks to me that we don't need a list here at all. Maybe just convert the whole section into paragraphs?

Reply:
Okay, I converted the list into a paragraph